diff --git a/include/hUGEDriver.h b/include/hUGEDriver.h
index 05badb4..72fb7b9 100644
--- a/include/hUGEDriver.h
+++ b/include/hUGEDriver.h
@@ -92,20 +92,20 @@ typedef struct hUGESong_t {
 } hUGESong_t;
 
 // initialize the driver with song data
-void hUGE_init(const hUGESong_t * song) OLDCALL;
+void hUGE_init(const hUGESong_t * song);
 void hUGE_init_banked(const hUGESong_t * song) BANKED OLDCALL;
 
 // driver routine
-void hUGE_dosound() OLDCALL;
+void hUGE_dosound();
 void hUGE_dosound_banked() BANKED OLDCALL;
 
 enum hUGE_channel_t {HT_CH1 = 0, HT_CH2, HT_CH3, HT_CH4};
 enum hUGE_mute_t    {HT_CH_PLAY = 0, HT_CH_MUTE};
 
-void hUGE_mute_channel(enum hUGE_channel_t ch, enum hUGE_mute_t mute) OLDCALL;
+void hUGE_mute_channel(enum hUGE_channel_t ch, enum hUGE_mute_t mute);
 void hUGE_mute_channel_banked(enum hUGE_channel_t ch, enum hUGE_mute_t mute) BANKED OLDCALL;
 
-void hUGE_set_position(unsigned char pattern) OLDCALL;
+void hUGE_set_position(unsigned char pattern);
 void hUGE_set_position_banked(unsigned char pattern) BANKED OLDCALL;
 
 extern volatile unsigned char hUGE_current_wave;
